i18n/translations/en.ts

```diff
@@ -260,13 +260,23 @@ const en = {
   // Group Invites
   group_admin_approval:
     "A group admin may need to approve your membership prior to joining.",
+  "A group admin may need to approve your membership prior to joining.":
```

Comment:
This feels like a weird approach, why use full sentences rather than keys?

Reply:
@thierryskoda and I discussed this approach and while at first glance it may seem strange, you should look at it from the perspective of the screen you're building / looking for a screen from a design/app run.

If I try and create a short key for the string, I have to search and find it in this file, and then jump to where that string is used in the code. That's more steps and those steps add up quick since string search is easiest way to find where in the app something is happening. 

cc @thierryskoda for thoughts

Reply:
Agree with Michael that this way worked best for me. Especially for finding screen.